app/src/test/java/com/github/sdp/mediato/modelTests.java

```diff
@@ -0,0 +1,51 @@
+package com.github.sdp.mediato;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+import static org.junit.Assert.*;
+
+import com.github.sdp.mediato.model.User;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class modelTests {
+
+    @Test
+    public void user_builder_registers_mandatory_attributes(){
+        //Build new user
+        User  user = new User.UserBuilder("uniqueId")
+                .setUsername("user")
+                .setDisplayedName("displayed name")
+                .setEmail("email")
+                .setRegisterDate("09/03/2023")
+                .setBirthDate("09/03/2023")
+                .setLocation(Arrays.asList(3.14, 3.14))
+                .build();
+
+        //Check values
+        Assert.assertEquals("uniqueId", user.getId());
+        Assert.assertEquals("user", user.getUsername());
+        Assert.assertEquals("displayed name", user.getDisplayedName());
+        Assert.assertEquals("email", user.getEmail());
+        Assert.assertEquals("09/03/2023", user.getRegisterDate());
+        Assert.assertEquals("09/03/2023", user.getBirthDate());
+        assertTrue(user.getLocation().containsAll(Arrays.asList(3.14, 3.14)));
+    }
+
+    @Test
+    public void user_builder_fails_with_invalid_strings(){

```

Comment:
Could you add a comment explaining what this test does? I'm not sure if this test is supposed to fail because there are missing attributes or because some strings are invalid.
We should also add tests that check that Preconditions fail on invalid strings but I guess that can wait for now as we haven't defined the formats yet.

Reply:
Thanks for pointing that out :) I clarified its name in commit [2d89969].
Yes the rest of the tests should be added when we clearly define the necessary formats (that's valid for all the model classes and their attributes, this is just a "draft" so that the methods can be used by the rest of the team during the sprints).